proto/expr.proto

```diff
@@ -117,6 +117,8 @@ message ExprNode {
     SQRT = 256;
     DEGREES = 257;
     RADIANS = 258;
+    ENCODE = 259;
+    DECODE = 260;
```

Comment:
Historically we have mixed both number-related and string-related expressions to 2xx. Could you put these 2 new string/bytea expressions starting 310 with a comment saying "string/bytea expressions"?